src/sentry/api/serializers/models/alert_rule_trigger.py

```diff
@@ -58,7 +58,7 @@ def get_attrs(self, item_list, user, **kwargs):
             exclusions.append(project_slug)
         return result
 
-    def serialize(self, obj, attrs, user):
+    def serialize(self, obj, attrs, user, **kwargs):
         data = super().serialize(obj, attrs, user)
```

Comment:
@NisanthanNanthakumar should `**kwargs` be passed to `super().serialize()` as well?

Reply:
👍 good catch!